CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 * `skbio.metadata.IntervalMetadata.drop` has a new boolean parameter `negate` to indicate whether to drop or keep the specified `Interval` objects.
 
+* `skbio.stats.distance` adding function permdisp. This fixes issue 1228
```

Comment:
this needs to be updated to 0.5.2-dev as we already released 0.5.2. update after #1565 gets merged.